### ISSUE TITLE
Docker PrivateBin error with exit code 1

### ISSUE THREAD

**Issue by nickshanks347** [PrivateBin/docker-nginx-fpm-alpine#47]
Trying to host PrivateBin on my RPi4, I get an error though and it exits with code 1:

```
privatebin     | [10-Jun-2021 00:27:29] NOTICE: fpm is running, pid 38
privatebin     | [10-Jun-2021 00:27:29] NOTICE: ready to handle connections
privatebin     | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
privatebin     | s6-svscan: warning: unable to iopause: Invalid argument
privatebin     | s6-svscan: warning: executing into .s6-svscan/crash
privatebin     | [s6-init] ensuring user provided files have correct perms...s6-svscan panicked! Dropping to a root shell.
privatebin     |
privatebin     | /bin/sh: can't access tty; job control turned off
privatebin     | /run/s6/services $
privatebin     | [10-Jun-2021 00:27:32] NOTICE: fpm is running, pid 43
privatebin     | [10-Jun-2021 00:27:32] NOTICE: ready to handle connections
privatebin     | [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
privatebin     | s6-supervise nginx: warning: can't happen: timeout while the service is up!
privatebin     | s6-supervise php-fpm8: warning: can't happen: timeout while the service is up!
privatebin     | s6-svscan: warning: unable to iopause: Invalid argument
privatebin     | s6-svscan: warning: executing into .s6-svscan/crash
privatebin     | [s6-init] ensuring user provided files have correct perms...s6-svscan panicked! Dropping to a root shell.
```

I am using docker-compose, it looks like this:

```
privatebin:
    image: privatebin/nginx-fpm-alpine:latest
    container_name: privatebin
    volumes:
      - ./privatebin:/srv/data
    ports:
      - "8085:8080"
```

**Comment by elrido** [comment]
This is very likely the same bug with the skalib lib compile options on alpine 3.13 for armv7 architecture, described in #27. This did get fixed in their edge, but not backported into the stable release. You can wait for a few more days, as the alpine 3.14 release is imminent and at that point we'll provide updated images for the stable 1.3.5 release or you can use our edge tag, which uses alpine:edge as the base of the image.

I'm actually very happy that so many people use the armv7 image - it shows us that the effort of maintaining it isn't in vain.

**Comment by nickshanks347** [comment]
Thanks @elrido, changing to edge worked. I'll keep an eye out on the tags so I can switch back to latest once alpine 3.14 is used for PrivateBin. 

As PrivateBin is so lightweight, I think it would be a lot of peoples preference to use it another lightweight device, such as a Raspberry Pi 👍

**Comment by elrido** [comment]
FYI, with alpine 3.14 released, we got our images rebuilt and the `1.3.5-alpine3.14`, `1.3.5` and `latest` tags now all contain the fixed s6 for armv7.

**Comment by nickshanks347** [comment]
> FYI, with alpine 3.14 released, we got our images rebuilt and the 1.3.5-alpine3.14, 1.3.5 and latest tags now all contain the fixed s6 > for armv7.

Thanks for the update @elrido.
